cmd/cluster-capi-operator/main.go

```diff
@@ -84,7 +84,7 @@ func initScheme(scheme *runtime.Scheme) {
 	utilruntime.Must(mapiv1beta1.AddToScheme(scheme))
 }
 
-//nolint:funlen
+//nolint:funlen,cyclop
```

Comment:
We should maybe think about separating some of the segments of the main into helper functions to make this easier to comprehend